src/Program.cs

```diff
@@ -19,18 +19,19 @@ public static void Main(string[] args)
             Host.CreateDefaultBuilder(args)
                 .ConfigureSecretStore((context, config, builder) =>
                 {
-                    builder.AddEnvironmentVariableProvider()
-                           .AddAzureKeyVaultProvider(
+                    builder.AddJsonFileSecrets("appsettings.Development.json")
```

Comment:
Frankly, I think we should just use a `AddConfiguration` so that we use whatever IConfiguration uses; less pain for us.

Reply:
Ah, yes, so they can build like their own `IConfiguration`.
Like:

```cshrarp
var config = new ConfigurationBuilder()
    .AddJsonFile("appsettings.Development.json");
builder.AddConfiguration(config);
```

It would be good that  the configuration is not yet created here, no? That's lazy loaded like the other secret providers. We can't use the `config` from the `ConfigureSecretStore((context, config, ...)` bc it might contain some other configuration values too, right?

Reply:
Actually the configuration is already built and passed here; so I should just be required to do `AddConfiguration` or `AddConfiguration(config)`; rest is handled already.

Reply:
So, you would use the already build `IConfiguration` here where possible other values (non secrets) are included too? 
It's not like you would be creating a `appsettings.Development.json` and a `appsettings.Secrets.json` or so?

Reply:
Personally I think we shouldn't reinvent things and just use, but if we do `AddConfiguration(config)` they can choose themselves - Either pass the current config or create one from scratch

Reply:
And maybe pass in `IConfigurationBuilder` instead of `IConfiguration` so we can lazy load it?

Reply:
The configuration is already provided here:
`.ConfigureSecretStore((context, config, builder)`

This needs to be `IConfiguration` so that we can build secret stores based on these.

Since we have `IConfiguration`, I would just use that.